packages/backend/src/api/controllers/EventsController.ts

```diff
@@ -0,0 +1,17 @@
+import { ProjectId } from '@l2beat/common'
+
+import { EventRepository } from '../../peripherals/database/EventRepository'
+
+export class EventsController {
+  constructor(private eventsRepository: EventRepository) {}
+
+  async getByProjectByEvent(projectId: ProjectId, eventName: string) {
```

Comment:
We also probably want to return the data in a format more like /api/main with arrays rather than objects. Again, could be another PR